docs/introduction/index.md

```diff
@@ -1,16 +1,41 @@
-3. download the results of the simulation, either every data file produced or just some specific files.
+In essence, the API enables you to:
 
-That’s it! Of course, there are many details and possible variations, but this is the basic flow of usage of the API. So, this section will give you an overview of how the API works, going into a bit more detail into how the API operates under the hood. 
+<div align="center">
+   <img src="./_static/infographic-apifunctionality-fullscreen.svg" alt="Inductiva API Usage Flow">
+</div>
 
-More specifically, we will introduce you to (Maya: what follows is basically what we have on the TOC):
 
-- Tasks. This is the core computational object of the platform. Your simulation requests are Tasks to be executed remotely on the computational infrastructure we make available
-- Shared and Dedicated Resources. Tasks get executed on computational resources, which can be Shared by all users, or which can be Dedicated to singe users
-- Storage and Data Flow. The platforms move a lot of data around. Your input files need to be sent to remote computational resources and the output of the simulations needs to be made available to you. 
-- Infrastructure. Under the hood, there is an infrastructure that provides computational power for executing your tasks. We currently use Google Cloud Platform, and very soon we will have other options available.
+1. Start and manage remote Virtual Machines (VMs), operating either independently 
+or collectively as an MPICluster, that are equipped with the pre-installed simulation 
+software of your choice.
+
+2. Send your simulation scripts from your laptop to one or more remote machines, 
+and start the simulators.
+
+3. Download simulation results, selecting either all data files generated or 
+specific ones of interest.
+
+That’s it! While this outlines the the basic flow of usage of the API, there are
+nuances and additional options available, which we'll explore in detail throughout
+this section. More specifically, you'll become familiarized with how the API functions 
+as we introduce you to:
+
+- **Tasks**: The core computational object of the platform, representing your 
+simulation requests to be processed remotely on the computational infrastructure 
+we make available.
+
+- **Shared vs. Dedicated Resources:** Understanding how tasks get executed on 
+computational resources that are either **shared** among all users or **dedicated** 
+to individual users.
+
+- **Storage and Data Flow:** Understanding the flow of data across platforms, from 
+sending your input files to remote computational resources to accessing the 
+simulation outputs.
+
+- **Infrastructure:** A further look under the hood, representing the underlying 
+infrastructure powering your simulations, currently leveraging [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-resource), with more options on the horizon.
```

Comment:
I would point to our new page about infra-structure, not outbound.
In fact, I think each of the these would point to the content that we have already there.
Or not, because we have the side navigation, but I would remove the outbound link.

Reply:
I think we can open a PR tomorrow just for links, the toc and the titles still need work imo.
I will link these to their respective places now